roles/KubevirtTemplateValidator/files/service.yaml

```diff
@@ -0,0 +1,89 @@
+---
+kind: ServiceAccount
+apiVersion: v1
+metadata:
+  name: template-validator
+  namespace: kubevirt
+  labels:
+    kubevirt.io: virt-template-validator
+---
+apiVersion: rbac.authorization.k8s.io/v1beta1
+kind: ClusterRoleBinding
+metadata:
+  name: template-validator
+  namespace: kubevirt
+  labels:
+    kubevirt.io: virt-template-validator
+roleRef:
+  kind: ClusterRole
+  name: template:view
+  apiGroup: rbac.authorization.k8s.io
+subjects:
+  - kind: ServiceAccount
+    name: template-validator
+    namespace: kubevirt
+---
+apiVersion: v1
+kind: Service
+metadata:
+  name: virt-template-validator
+  namespace: kubevirt
+  labels:
+    kubevirt.io: virt-template-validator
+  annotations:
+    service.alpha.openshift.io/serving-cert-secret-name: virt-template-validator-certs
+spec:
+  ports:
+  - name: webhook
+    port: 443
+    targetPort: 8443
+  selector:
+    kubevirt.io: virt-template-validator
+---
+apiVersion: apps/v1beta1
+kind: Deployment
+metadata:
+  name: virt-template-validator
+  namespace: kubevirt
+  labels:
+    name: virt-template-validator
+spec:
+  replicas: 1
+  template:
+    metadata:
+      name: virt-template-validator
+      labels:
+        kubevirt.io: virt-template-validator
+    spec:
+      serviceAccountName: template-validator
+      containers:
+        - name: webhook
+          image: quay.io/fromani/kubevirt-template-validator:v0.3.0
```

Comment:
Can we extract this to a variable (role default)? Just so we can make downstream builds easier.